tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "target": "es6",
     "strict": true,
     "noImplicitAny": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
I understand why you need this change, but I'm wondering if there is another way to fix the problem.. adding `generated` folder to _exclude_ list didn't make any difference..

Reply:
I made this change because there was no other way to ignore generated folder errors. Throwing this kind of warning as an error is "pretty useless" because from a code perspective (not generated code) we have eslint that informs us about it.

Reply:
Restored in  https://github.com/pagopa/io-backend/pull/835/commits/8750a4e17df0b6ce831285482da1c50ada93bff0 after an update of the codegen.